ports/glutin/browser.rs

```diff
@@ -401,6 +401,10 @@ where
                         self.event_queue.push(WindowEvent::SendError(None, reason));
                     };
                 },
+                EmbedderMsg::PromptPermission(message, dialog_title, sender) => {
```

Comment:
If you build `./mach build -d --libsimpleservo`, you will see code in ports/libsimpleservo/api and ports/libsimpleservo/capi that need to handle this (by invoking a new method in HostTrait), and you can add stub implementations to ports/libsimpleservo/jniapi and ports/libmlservo for this new method as well.